packages/web/src/pages/track-page/components/shared/RemixContestWinnersTab.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTrack } from '@audius/common/api'
+import { ID } from '@audius/common/models'
+
+import { trackRemixesPage } from 'utils/route'
+
+import { RemixSubmissionCardSize } from './RemixGrid'
+import { RemixTab } from './RemixTab'
+
+const messages = {
+  noWinners: 'No winners announced yet',
+  stayTuned: 'Stay tuned for the results!',
+  viewAll: 'View All'
+}
+
+type RemixContestWinnersTabProps = {
+  trackId: ID
+  winnerIds: ID[]
+  size?: RemixSubmissionCardSize
+}
+
+/**
+ * Tab content displaying winners for a remix contest
+ */
+export const RemixContestWinnersTab = ({
+  trackId,
+  winnerIds,
+  size = 'desktop'
+}: RemixContestWinnersTabProps) => {
+  const { data: permalink } = useTrack(trackId, {
+    select: (track) => track.permalink
+  })
+
+  const pathname = trackRemixesPage(permalink ?? '')
```

Comment:
on local dev sometimes i was getting heavy load screen from a failed track fetch, wondering if something like this would be the cause? bc is it possible we're fetching with trackId='' sometimes?